server_installation/topics/operator/installation.adoc

```diff
@@ -0,0 +1,68 @@
+
+=== Installing {project_operator} into OpenShift cluster
+
+There are ways of running {project_operator} in Kubernetes/OpenShift cluster:
```

Comment:
@slaskawi Maybe missing `two` above? Like `There are two ways of running..`

Reply:
Fixed